components/namespace-lister/production/kflux-prd-rh02/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../../base/
```

Comment:
I'd suggest to use a separate `base` under production so we don't share manifests between environments

Reply:
Fixed in d84b890f.